test/unit/uk/gov/hmrc/apisubscriptionfields/service/FieldsDefinitionServiceSpec.scala

```diff
@@ -50,34 +50,34 @@ class FieldsDefinitionServiceSpec extends UnitSpec with FieldsDefinitionTestData
       result shouldBe FieldsDefinitionResponse(Seq(FakeFieldDefinitionUrl, FakeFieldDefinitionString))
     }
 
-    "return None when no entry exist in the repo when get by identifier is called" in {
-      (mockFieldsDefinitionRepository fetch (_: FieldsDefinitionIdentifier)) expects FakeFieldsDefinitionIdentifier returns None
+    "return None when no entry exist in the repo when get by fields definition is called" in {
```

Comment:
`return None when no definition exists in the repo`

Reply:
Done